broker-cli/commands/wallet.js

```diff
@@ -290,6 +329,111 @@ const CAPACITY_STATUSES = Object.freeze({
+
+  statusTable.push(['Outstanding', '', ''])
+  statusTable.push([`  Buy ${baseSymbol}`, formatBalance(baseSymbolCapacities.outstandingReceiveCapacity, NETWORK_STATUSES.OUTSTANDING), formatBalance(counterSymbolCapacities.outstandingSendCapacity, NETWORK_STATUSES.OUTSTANDING)])
+  statusTable.push([`  Sell ${baseSymbol}`, formatBalance(baseSymbolCapacities.outstandingSendCapacity, NETWORK_STATUSES.OUTSTANDING), formatBalance(counterSymbolCapacities.outstandingReceiveCapacity, NETWORK_STATUSES.OUTSTANDING)])
+
+  statusTable.push(['Pending', '', ''])
+  statusTable.push([`  Buy ${baseSymbol}`, formatBalance(baseSymbolCapacities.pendingReceiveCapacity, NETWORK_STATUSES.PENDING), formatBalance(counterSymbolCapacities.pendingSendCapacity, NETWORK_STATUSES.PENDING)])
+  statusTable.push([`  Sell ${baseSymbol}`, formatBalance(baseSymbolCapacities.pendingSendCapacity, NETWORK_STATUSES.PENDING), formatBalance(counterSymbolCapacities.pendingReceiveCapacity, NETWORK_STATUSES.PENDING)])
+
+  statusTable.push(['Inactive', '', ''])
+  statusTable.push([`  Buy ${baseSymbol}`, formatBalance(baseSymbolCapacities.inactiveReceiveCapacity, NETWORK_STATUSES.INACTIVE), formatBalance(counterSymbolCapacities.inactiveSendCapacity, NETWORK_STATUSES.INACTIVE)])
+  statusTable.push([`  Sell ${baseSymbol}`, formatBalance(baseSymbolCapacities.inactiveSendCapacity, NETWORK_STATUSES.INACTIVE), formatBalance(counterSymbolCapacities.inactiveReceiveCapacity, NETWORK_STATUSES.INACTIVE)])
+
+  logger.info(` Market: ${market.bold.white}`)
+  logger.info(statusTable.toString())
+  logger.info('')
+
+  if (baseSymbolCapacities.status !== CAPACITY_STATUSES.OK) {
+    logger.error(`${baseSymbol}: Received errors when requesting network status: ${baseSymbolCapacities.error}`.red)
+  }
+
+  if (counterSymbolCapacities.status !== CAPACITY_STATUSES.OK) {
+    logger.error(`${counterSymbol}: Received errors when requesting network status: ${counterSymbolCapacities.error}`.red)
+  }
+}
+
+/**
+ * Prints network status summary in JSON format.
+ * @param {String} market
+ * @param {Object} baseSymbolCapacities
+ * @param {Object} counterSymbolCapacities
+ * @param {Object} logger
+
+ * @returns {Void}
+ */
+
+function networkStatusJSON (market, baseSymbolCapacities, counterSymbolCapacities, logger) {
+  const baseSymbol = baseSymbolCapacities.symbol.toUpperCase()
+  const counterSymbol = counterSymbolCapacities.symbol.toUpperCase()
+
+  // If any balances in the following table are empty (which occurs if the engine is unavailable)
+  // then the text will show up as `Not Available`.
+  //
+  // The user will then be prompted with a warning letting them know that we failed
+  // to receive a balance for a particular currency
+  const networkStatus = {
+    market: market,
+    baseSymbol: baseSymbol,
+    counterSymbol: counterSymbol,
+    statuses: [
```

Comment:
I'm not sure that an array is the best format for this, a straight object might be better, e.g.

```
...
statuses: {
  available: {
    sendCapacity:
...
```

Or better yet, why not format it more like the actual return value of the RPC?